site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,435 @@
+
+As a logged in user, you'll receive server pushes whenever a chat or a thread is started, closed, etc. It applies to both the Agent and Customer Chat API. 
+
+The [incoming event](../agent-chat-api/#incoming-event) push will inform you about the events sent to the chat (on both the agent and customer side).
+
+
+## Properties
+
+> Sample **properties**: 
+
+```js
+{
+    "properties": {
+        "routing": {
+            "pinned": true,
+            "count": 3
+        }
+    }
+}
+```
+Properties are key-value storages. They can be set within a chat, a thread, or an event. 
+
+In our example, `routing` is the namespace, while `pinned` and `count` are properties names.
+
+
+### Configuration
+
+> **Example: using properties to create a basic chat rating.** 
+
+> We need two properties: `rating_score` and `rating_comment`. They should be writable by a Customer, and readable by an Agent in a chat.
+
+> First, we create our properties configuration using the Configuration API.
+
+```json
+curl -v https://api.livechatinc.com/configuration/properties/create_properties \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer c5e4f61e1a6c3b1521b541bc5c5a2ac5" \
+    -X POST -d '
+{
+    "rating_score" : {
+        "type" : "int",
+        "locations" : {
+            "chat" : {
+                "access" : {
+                    "agent" : {
+                        "read" : true,
+                        "write" : false
+                    },
+                    "customer" : {
+                        "read" : true,
+                        "write" : true
+                    }
+                }
+            }
+        }
+    },
+    "rating_comment" : {
+        "type" : "string",
+        "locations" : {
+            "chat" : {
+                "access" : {
+                    "agent" : {
+                        "read" : true,
+                        "write" : false
+                    },
+                    "customer" : {
+                        "read" : true,
+                        "write" : true
+                    }
+                }
+            }
+        }
+    }
+}'
+```
+<!-- Description -->
+You can create properties within a licence and configure them using the [Configuration API](../configuration-api/). They are grouped in namespaces, which helps distinguishing which property belongs to a given integration. Your namespace is always named after your `application id`.
+
+You can configure the property [type](#property-types), [location](#property-locations), and [domain](#property-domain).
+
+#### Property types
+
+There are four property types:
+
+- `int` (int32)
+- `bool`
+- `string`
+- `tokenized_string`
+
+The `tokenized_string` type is a string split to tokens before indexing in our search engine. It can be useful when using a property as a filter in certain methods, for example in [get_archives](../agent-chat-api/#get-archives).
```

Comment:
może być użyteczne przy dłuższych tekstach, jak np. tekst wiadomości. Nie używac do słów kluczowych.

Reply:
I don't know if we should describe properties in detail in this doc. I think there should only be a short desc (like for webhooks) and the rest should be moved to other documents

Reply:
For now, moved to Agent Chat API. Can be copy-pasted to Configuration API